### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by kurbata**

### CODE REVIEW

**Review by ievgendco — CHANGES_REQUESTED** [review]
Too much files was added

File: cypress/Integrations/ToolsQAShop.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Feature: Shopping on Tools QA
```
Comment (ievgendco) [inline cypress/Integrations/ToolsQAShop.feature]: Are you sure you are using BDD solution for your tests?

File: cypress/e2e/sign_in_page.spec.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+const {sign_in_page} = require("../Selectors/sign_in_page");
+
+describe('UI tests for sign in page', () => {
+
```
Comment (ievgendco) [inline cypress/e2e/sign_in_page.spec.js:4]: Should be a single space between blocks

File: cypress/e2e/sign_in_page.spec.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    it('should show "Sign in" title', () => {
+        cy.get(sign_in_page.title_text).should('be.visible').and('have.text', 'Sign in')
+    })
+
+    // todo: automate foloowing test cases:
+    it('should show typeable Username field', () => {
+        cy.get(sign_in_page.input_user_name_field).should('be.visible').type('user name')
+    })
+    it('should show typeable Password field', () => {
```
Comment (ievgendco) [inline cypress/e2e/sign_in_page.spec.js:21]: See comment above

**Review by ievgendco — CHANGES_REQUESTED** [review]

File: cypress/e2e/sign_in_page.spec.js (hunk 3 above)
Comment (kurbata) [reply cypress/e2e/sign_in_page.spec.js:21]: Fixed

File: cypress/e2e/sign_in_page.spec.js (hunk 2 above)
Comment (kurbata) [reply cypress/e2e/sign_in_page.spec.js:4]: Fixed

**Pull request merged**